src/main/resources/vertx-maven-plugin.properties

```diff
@@ -1,3 +1,5 @@
 maven-compiler-plugin-version=3.1
```

Comment:
Why do we have all the plugin version ? Should be vertx-core and the vertx-maven-plugin version enough ?